app/views/articles/tag_index.html.erb

```diff
@@ -4,31 +4,34 @@
 <% expiry_minutes = params[:timeframe].blank? || params[:timeframe] == "latest" ? 4 : 20 %>
 <% flag = true %>
 <% cache("tag-stories-index-#{params}-#{flag}-#{@tag_model.updated_at}-#{user_signed_in?}", expires_in: expiry_minutes.minutes) do %>
-  <style>
-    .tag-header {
-      border: 2px solid<%= HexComparer.new([@tag_model.bg_color_hex || "#0000000", @tag_model.text_color_hex || "#ffffff"]).brightness(0.88) %>;
-      box-shadow: 5px 6px 0px<%= HexComparer.new([@tag_model.bg_color_hex || "#0000000", @tag_model.text_color_hex || "#ffffff"]).brightness(0.88) %>;
-    }
-  </style>
-  <div class="user-profile-header tag-header">
-    <div class="tag-or-query-header-container">
-      <h1>
-        <% if @tag_model.badge_id && @tag_model.badge %>
-            <img src="<%= optimized_image_url(@tag_model.badge.badge_image_url, width: 180) %>" style="transform: rotate(<%= -25 + (@tag_model.id.digits.first * 5) %>deg);" />
-        <% end %>
-        <% if @tag_model && @tag_model.pretty_name.present? %>
-          <%= @tag_model.pretty_name %>
-        <% else %>
-          <%= @tag %>
-        <% end %>
+  <div class="crayons-layout">
+    <header class="tag-header crayons-card branded-4 relative" style="border-top-color: <%= HexComparer.new([@tag_model.bg_color_hex || "#0000000", @tag_model.text_color_hex || "#ffffff"]).brightness(0.88) %>">
+      <div class="tag-header__title flex justify-between items-center mb-2 s:mb-4">
```

Comment:
Instead of having `mb-2 s:mb-4` here, I'd move it to the paragraph on line 28 as a `pt-2 s:pt-4` - therefore if there's no `short_summary`, it will be better aligned:

**With `mb...`** 

![image](https://user-images.githubusercontent.com/108287/97418173-c9bd6f00-1908-11eb-8314-6b97b1d5ae89.png)

**With `pt...`**

![image](https://user-images.githubusercontent.com/108287/97418243-e0fc5c80-1908-11eb-8506-06b9a4a8cabb.png)

Reply:
You don't seem to be referencing `tag-header__title` anywhere in CSS or JS so I think we can drop it, right?